analytics/contentsquare/experience-accelerator/src/CSClient.ts

```diff
@@ -0,0 +1,30 @@
+import { Client } from "./Client";
+
+export class CSClient extends Client {
+    constructor(
+      public readonly maxWaitTime = 5000
+    ) {
+        super(maxWaitTime);
+    }
+
+    sendMetrics(type: string, event: any) {
+        var uid = this.getUid(event);
+        let cidEid = this.getCidEid(event);
+
+        this.pushCSEvent(uid, cidEid);
+    }
+
+    pushCSEvent(userId: string, cidEid: string) {
+        // CS integration should resplit the eid and cid
+        var cidEidArray = cidEid.split(':', 2);
+        var cid = cidEidArray[0];
+        var eid = cidEidArray[1];
```

Comment:
This won't be able to distinguish between cloned cids as is chops off the random number in the 3rd part